cockatrice/src/chatview/chatview.cpp

```diff
@@ -93,7 +93,7 @@ void ChatView::appendCardTag(QTextCursor &cursor, const QString &cardName)
     anchorFormat.setAnchor(true);
     anchorFormat.setAnchorHref("card://" + cardName);
     anchorFormat.setFontItalic(true);
-    
+
```

Comment:
Tip: When making changes, use `git add -p` to prepare your commit diff by diff. That way you can sculpt specific commits, and do the behavior changes in one, and formatting/refactoring in another